homeassistant/components/reddit/sensor.py

```diff
@@ -139,5 +156,5 @@ def update(self):
                         }
                     )
 
-        except praw.exceptions.PRAWException as err:
+        except PRAWException as err:
```

Comment:
How is the change of importing a typing change?
And why is the exception imported directly, but `praw.Reddit` isn't?